### ISSUE TITLE
MysqlSubscription with Router

### ISSUE THREAD

**Issue by tommyJS** [numtel/meteor-mysql#20]
Is it possible to use MysqlSubscription with Iron-Router or Flow-Router?

**Comment by VCarrasco** [comment]
Yes, You can use MysqlSubscription with Iron Router instead of using it on the client file. 

Router.route('page', {
...  
subscriptions: function() {
      return [ 
           subsctriptionOne = new MysqlSubscription('name_one'), 
           subsctriptionTwo = new MysqlSubscription('name_two')
      ]
...
} );

**Comment by sushitommy** [comment]
Hmm, I can't get a MysqLSubscription working in Iron-router.
Keep getting the error "Exception in callback of async function: Error: Subscription failed!"

Router.route('pageDetail', {
  path: '/pages/:_id',
  subscriptions: function(){
    return new MysqlSubscription('data', 'dDB2xAbbSkG4HJCvW');
  }
});

Defining the mysqlSubscription outside of iron router works perfectly. What could be the problem?

**Comment by sushitommy** [comment]
I cant get this to work, I keep getting the error "Exception in callback of async function: Error: Subscription failed!".

My code:

Router.route('measurementDetail', {
  path: '/measurements/:_id/dashboard',
  waitOn:function(){
    return [
      mysqlSub = new MysqlSubscription('measurementData', this.params._id)
    ]
  }
});

Why doesn't this work?

**Comment by kiyasov** [comment]
HELP

**Comment by jacasanov** [comment]
This work:
```
waitOn: function() {
	return [Subs1.reactive(), Subs2.reactive(), Subs3.reactive()];
},
```
